config/scripts.yml

```diff
@@ -956,3 +956,15 @@ dockless_pt_poly:
   job: true
   path: ../transportation-dockless-processing
   source: mds
+email_service_request:
+  args:
+    - dts_portal_prod
+  cron: '*/15 * * * *'
```

Comment:
@JoeyL6 this cron schedule is set for every 15 minutes. we discussed running this more frequently. e.g. every minute?

Reply:
My bad, I've changed the frequency to every minute.